lib/templates/android_screen_base.tt

```diff
@@ -183,4 +183,23 @@ class AndroidScreenBase < Calabash::ABase
     # Clicking in the Done button
     touch "* id:'button1'"
   end
+  
+ 
+  
+  # @option [String] id The view's id (optional)
+  # @option [String] The complete view query
+  # @return [String] returns a string associated with a single view, if the case
+  # @return [Array] return an array[String] associated with the views, if the case
+  # Example ->     get_text_from(query_string_variable)
+  def get_text_from(id = nil, query)
+    query_text = "* id:'#{id}'" unless id.nil?
+    query_text = query unless query.nil?
+    array_text = query(query_text, :text)
+
+    # returns an array[String] or text dependending of the quantity of views found
+    return array_text.first unless array_text.size > 1
```

Comment:
@CSRafaelAlves I was thinking about this method. I think that this two types of return will make harder to use it. What about create two methods?

The first will always return an array of texts
```
def text_array_from(id = nil, query)
  query_text = "* id:'#{id}'" unless id.nil?
  query_text = query unless query.nil?
  array_text = query(query_text, :text)

  array_text
end
```

The second one, will always return a text
```
def text_from(id = nil, query)
  text_array_from(id, query).first
end
```

One last thing, get_* is not a standard method name in ruby, the usual would be to not use get or set in method names, so I used another names, tell me what you think about it.